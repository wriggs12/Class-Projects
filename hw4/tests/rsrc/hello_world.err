[00000.000000] STARTUP
[00000.000033] PROMPT
[00000.100226] INPUT run echo hello world
[00000.100395] CHANGE 2186481: none -> running
[00000.100953] SIGNAL 17
[00000.100973] CHANGE 2186481: running -> stopped
[00000.101008] PROMPT
[00000.201124] INPUT cont 0
[00000.201162] CHANGE 2186481: stopped -> running
[00000.201183] PROMPT
hello world
[00000.202169] SIGNAL 17
[00000.202196] CHANGE 2186481: running -> dead
[00000.301382] INPUT quit
[00000.301391] SHUTDOWN
