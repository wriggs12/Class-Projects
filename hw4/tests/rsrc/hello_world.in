run echo hello world
cont 0
quit
