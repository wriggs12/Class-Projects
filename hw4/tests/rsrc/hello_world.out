0	2186481	T	running		echo hello world
0	2186481	T	stopped		echo hello world
0	2186481	T	running		echo hello world
0	2186481	T	dead	0x0	echo hello world
